From 2fd7b1f22d2e03dac423adace92ab2214305b4ac Mon Sep 17 00:00:00 2001
From: Jerome Forissier <jerome@forissier.org>
Date: Tue, 26 May 2020 18:16:16 +0200
Subject: [PATCH 5/6] [Fixup] Allow use of cc-option in core/arch/arm/arm.mk

Fix a build error, let's see if Shippable is happy with this, I
still do not understand why these lines are needed.

Signed-off-by: Jerome Forissier <jerome@forissier.org>
Upstream-Status: Accepted [https://github.com/OP-TEE/optee_os/pull/3891]
Signed-off-by: Joshua Watt <JPEWhacker@gmail.com>
---
 core/core.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/core.mk b/core/core.mk
index 1a330457..d6af1d51 100644
--- a/core/core.mk
+++ b/core/core.mk
@@ -16,6 +16,10 @@ PLATFORM_FLAVOR_$(PLATFORM_FLAVOR) := y
 $(eval $(call cfg-depends-all,CFG_PAGED_USER_TA,CFG_WITH_PAGER CFG_WITH_USER_TA))
 include core/crypto.mk
 
+# Setup compiler for this sub module
+COMPILER_$(sm)         ?= $(COMPILER)
+include mk/$(COMPILER_$(sm)).mk
+
 cppflags$(sm)	+= -D__KERNEL__
 
 cppflags$(sm)	+= -Icore/include
-- 
2.17.1

